.github/workflows/nightly-rosetta-pax-build.yaml

```diff
@@ -123,49 +150,45 @@ jobs:
         ${{ needs.arm64.outputs.DOCKER_TAG_FINAL }}
       TARGET_IMAGE: pax
       TARGET_TAGS: |
-        type=raw,value=latest,priority=1000
-        type=raw,value=nightly-${{ needs.metadata.outputs.BUILD_DATE }},priority=900  
+        ${{ needs.test-amd64.outputs.TEST_STATUS == 'success' && 'type=raw,value=latest,priority=1000' || '' }}
+        type=raw,value=nightly-${{ needs.metadata.outputs.BUILD_DATE }},priority=900
 
-  test-pax:
-    needs: [metadata, amd64, arm64]
-    uses: ./.github/workflows/_test_pax_rosetta.yaml
-    if: (github.event_name == 'workflow_run' && github.event.workflow_run.conclusion == 'success') || github.event_name == 'workflow_dispatch'
-    with:
-      PAX_IMAGE: ${{ needs.amd64.outputs.DOCKER_TAG_FINAL }}
-    secrets: inherit
-
-  publish-test:
-    needs: [metadata, amd64, arm64, test-pax]
+  # TODO: ARM Tests
+  publish-test-badge:
+    needs: [metadata, publish-build-badge, test-amd64]
     uses: ./.github/workflows/_publish_badge.yaml
```

Comment:
Note that `_publish_badge.yaml`` will be deprecated soon.

Reply:
Would it be okay to tackle this in a subsequent PR? My thinking is we can get the reproducible build/ci changes out and then circle back to sitrep. I expect the reproducible PRs to all be flushed out within a week or two :)